testdata/scripts/literals.txt

```diff
@@ -220,6 +220,9 @@ func constantTest() {
 	const impType = imported.ImportedType(3)
 }
 
+// TODO: this only tests, that we don't break byte slices 
+// it was verified manually that these actually get obfuscated,
+// however we should add regression tests 
```

Comment:
please make this complete sentences, otherwise it's hard to tell if this entire paragraph is a single sentence or not.

for example, also with more detail:

```
// TODO: This only tests that we don't break byte slices.
// It was manually verified that they do get obfuscated,
// The original bytes don't seem to show up in the binary,
// meaning that we can't test for them via binsubstr.
// We should figure out a way to test for the byte sequences.
// For now, we manually tested these when they got added.
```